docs/how-to/push-update-on-mainnet.md

```diff
@@ -137,7 +137,7 @@ The following variables will be used in all following commands:
 CONFIG_KEY=mainnet_dryrun_full  # Should be the same as on the running nodes!
 RELAY_PEER=<IP>:3000            # A host of any relay node.
 
-COMMONOPTS="--system-start 0 --configuration-file ../cardano-sl/node/configuration.yaml --configuration-key ${CONFIG_KEY}"
+COMMONOPTS="--system-start 0 --configuration-file ../cardano-sl/lib/configuration.yaml --configuration-key ${CONFIG_KEY} --mode=with-config"
```

Comment:
Is with-config sufficient? Don't you need with-node to keep up with the current epoch?

Reply:
All previous examples do not have `--node-enabled` flag, so yes, `with-config` should be sufficient.

Reply:
At the time of writing the old version of the document I am not sure if `--node-enabled` existed. I'd like @gromakovsky's opinion on whether `with-node` would be better for testing updates (I think yes, because then you have up-to-date blockchain)

Reply:
The goal of this issue is to fix outdated parts of some documents. It should be discussed with DevOps whether we want to use `with-node` mode. Currently there are no benefits in doing it, because versions should be provided manually anyway. We can implement better interface which will set proper versions automatically (in `with-node` mode), but it will require keeping auxx running which might be not convenient.